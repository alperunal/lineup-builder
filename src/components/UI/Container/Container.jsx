import React from 'react';
import './Container.module.scss';

const Container = ({ children }) => <div className="container">{children}</div>;

export default Container;
