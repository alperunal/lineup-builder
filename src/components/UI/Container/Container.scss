.container {
  max-width: 1000px;
  width: 100%;
  margin: auto;
  padding: 0 20px;
}
