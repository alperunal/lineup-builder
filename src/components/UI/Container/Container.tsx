import React from 'react';
import styled from 'styled-components';

const Container = ({ children }) => {
    return <ContainerDiv>{children}</ContainerDiv>;
};

const ContainerDiv = styled.div`
    max-width: 900px;
    width: 100%;
    margin: auto;
`;

export default Container;
