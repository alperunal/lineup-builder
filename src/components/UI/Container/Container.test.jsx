import React from 'react';
import { shallow } from 'enzyme';
import Container from './Container';

it('renders Container correctly', () => {
  const wrapper = shallow(<Container>Test</Container>);

  expect(wrapper).toMatchSnapshot();
  expect(wrapper.contains('Test')).toBe(true);
});
