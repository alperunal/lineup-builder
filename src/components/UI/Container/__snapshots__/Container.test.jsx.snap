// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Container correctly 1`] = `ShallowWrapper {}`;
